8-CSS-Position-and-Transform/Part-1/css/style.css

```diff
@@ -0,0 +1,218 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.background {
+  z-index: -2;
+  position: fixed;
```

Comment:
it's shoudl be fixed cause in a while we will continue work on that whole mockup and when you will scroll it you will not see new elements cause this element is fixed

Reply:
am unlock him in the future